CMakeLists.txt

```diff
@@ -148,6 +151,10 @@ if(WITH_STL)
 endif()
 
 option(WITH_OTLP "Whether to include the OpenTelemetry Protocol in the SDK" OFF)
+
+option(WITH_OTLP_HTTP "Whether to include the OTLP http exporter in the SDK"
+       OFF)
```

Comment:
So to confirm, `WITH_OTLP_HTTP` is now a valid CMake option for command-line, but `WITH_OTLP_GRPC` is not. And this needs to be fixed as separate bug?